optimum/exporters/onnx/model_configs.py

```diff
@@ -550,6 +550,11 @@ def inputs(self) -> Dict[str, Dict[int, str]]:
         return {"pixel_values": {0: "batch_size", 1: "num_channels", 2: "height", 3: "width"}}
 
 
+class CvTOnnxConfig(ViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 13
+    ATOL_FOR_VALIDATION = 1e-2
```

Comment:
This seems pretty high tbh, do you have any idea which op makes it fail?

Reply:
I am talking about the tolerance, we might have a hidden bug if we need `1e-2`, or we might not, but just wanted to raise the potential issue here.